src/main.cpp

```diff
@@ -39,17 +42,32 @@ int main(int argc, char *argv[]){
     int length = 8;
 
     // set symbol set
-    bool symbol_set = false;
+    bool symbol_set = false,save_pass=false;
+
+    //set file path for writing password to current directory as default.
+    std::string file_path ;
 
     for(int i = 1; i<argc; i++) {
-        if(strcmp(argv[i], "-S") == 0) {
+        if(strcmp(argv[i],"-F")==0){
+            if(!save_pass){
+                save_pass = true;
+                file_path =argv[i+1];
+                i++;
```

Comment:
```suggestion
```

Reply:
Sir, i did this so that the file path for -F option is not treated as an unknown option.
I have now removed this and put a check in line number 69 instead.